source/ShipInfoDisplay.h

```diff
@@ -37,7 +37,7 @@ class ShipInfoDisplay : public ItemInfoDisplay {
 	ShipInfoDisplay(const Ship &ship, const Depreciation &depreciation, int day);
 
 	// Call this every time the ship changes.
-	void Update(const Ship &ship, const Depreciation &depreciation, int day);
+	void Update(const Ship &ship, const Depreciation &depreciation, int day, bool sale);
```

Comment:
how about making sale default on true since thats how we often use it?

also idk about sale mb it doesnt convey enough meaning... should be fine though

Reply:
It might make more sense for it to default to false, actually, since it is used more _not_ in the shipyard than in it.

Reply:
Also, I believe a parameter named sale is used to check for the same thing in other functions in the class.

Reply:
Alr default to false then if you feel like it, I just meant a default would make sense
Add a comment to say what sale means right above the function, too